kas/opt/pg2.yml

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Su Baocheng <baocheng.su@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+header:
+  version: 10
+
+local_conf_header:
+  board_variant: |
+    IOT2050_VARIANTS ?= "pg2"
```

Comment:
One of these option files is not needed.

Reply:
The one for PG2 is deleted.